exercises/E-conditionals/exercise.js

```diff
@@ -14,3 +14,14 @@ EXPECTED RESULT
 ---------------
 Hi, I'm Daniel, I'm a mentor.
 */
+
+function isDanielMentor(name, danielsRole){
+
+  if(danielsRole === "mentor"){
+  return  `Hi, I'm Daniel, I'm a mentor`;
+  }
+  else{
+    return `Hi, I'm Daniel, I'm a student`;
+  }
```

Comment:
It's great to see that you've learned how to use conditional 👏 

The goal of the functions is to use parameters passed to the functions so that we can run the function for many different names and roles.
For example, if I want to call a function for another mentor whose name is not Daniel, this function has a couple of things that need to be fixed:

1. Function name: `isDanielMentor` is only useful if the mentor name is Daniel, what about other mentors? To fix this, we pick a function name that is generic, e.g. `isMentor`

2. Parameter name: `danielsRole`. Same as above this will only work for Daniel, not others. It should be a generic name like `role`.

3. We've got the parameter `name` in the function. Any developer should be able to call this function with a name they have (e.g. Rahwa), and they want to be able to get results with their name. To fix this, we should avoid hardcoding the names into the strings in lines 21 and 24, instead, we can rewrite them to be like this:
```
return `Hi, I'm ${name}, I'm a mentor`;
```